scripts/nebula.service

```diff
@@ -141,6 +141,8 @@ function start_daemon {
     local command="${executable} --flagfile ${config}"
     INFO "Starting ${daemon_name}..."
     eval ${command}
+    local port=$(get_port_from_config ${config})
+    wait_for_port_listening ${port} 20 || WARN "Timed out waiting for ${daemon_name} to finish starting, but it may still be running"
```

Comment:
prefer to implement this logic in nebula process.
and the behavior should be same with running in container.

Reply:
or just like readinessprobe in k8s, curl a http interface?